app/build.gradle.kts

```diff
@@ -89,6 +89,6 @@ dependencies {
     implementation(libs.ktor.cio)
     implementation(libs.taglib)
     implementation(libs.datastore.preferences)
-    debugImplementation(libs.ui.tooling)
-    debugImplementation(libs.ui.tooling.preview)
+    implementation(libs.ui.tooling) //NOT RECOMMENDED
+    implementation(libs.ui.tooling.preview) //NOT RECOMMENDED
```

Comment:
if not recommended then why?

Reply:
It's not recommended because we are adding some Preview and Tooling things to the final release. This does not impact the performance in any way, but since we are not using a modularized project structure, we cannot delete that.

This is due to the fact that I created some component previews in the new components